addons/calendar/models/calendar_event.py

```diff
@@ -1184,7 +1184,8 @@ def ics_datetime(idate, allday=False):
             if not is_html_empty(meeting.description):
                 if 'appointment_type_id' in meeting._fields and self.appointment_type_id:
                     # convert_online_event_desc_to_text method for correct data formatting in external calendars
-                    event.add('description').value = self.convert_online_event_desc_to_text(meeting.description)
+                    discuss_url = f"Join with odoo discuss: {meeting.videocall_location}\n" if meeting.videocall_location else ""
+                    event.add('description').value = discuss_url + self.convert_online_event_desc_to_text(meeting.description)
```

Comment:
Hello @adsh-odoo 

As you can see on github, this code has changed since you implemented it.
There is now a "_get_customer_description" method that is overridden inside "appointment".

What I suggest is to use this "_get_customer_description" and modify it inside the calendar app to include the videocall link at the top of the description in the returned value.

And then use that "_get_customer_description" method when synchronizing with google / microsoft calendars instead of just using the "description" field.

It should work, BUT

I see that google needs to received sanitized HTML, and the "_get_customer_description" method converts to plaintext.

So you will need to do a preliminary commit where you modify the "_get_customer_description" to make it return HTML instead.
And in the existing calls to that method, you will need to wrap with "html2plaintext" manually.

That also means that you will have to modify the appointment override to insert "br" tags instead of "\n" for new lines, but it should not be too complicated.

It will in turn allow you to call that method when sending values to google.

Tell me if anything is unclear.

Thanks & cheers!